### ISSUE TITLE
wizardly doesn't work with rails 3.0.0rc

### ISSUE THREAD

**Issue by paulschreiber** [jeffp/wizardly#15]
wizardly doesn't work with rails 3. there are a few issues:
- the generators don't work [you can work around this]<br>\* validation_group doesn't work

I'm not sure if there are problems in wizardly itself, because I'm blocked on validation_group. I tried hacking on that, and could get it to compile, but no luck.

Has anyone successfully ported this to rails 3?

**Comment by roja** [comment]
any update on this?

**Comment by manuelmeurer** [comment]
Just trying to get Wizardly working in a Rails 3 app and using jfelchner's fork, he cleaned up some Rails 3 related issues.

**Comment by mhuggins** [comment]
Thanks for the heads up about jfelchner's fork, manuelmeurer.  Hopefully we find this merged into jeffp's branch at some point.

**Comment by shedd** [comment]
Any further updates on this?

**Comment by evolve2k** [comment]
updates?

**Comment by paulschreiber** [comment]
On Oct 31, 2011, at 3:29 AM, evolve2k
reply@reply.github.com
wrote:

> updates?

I have no news. There's a fork that someone made for rails 3, but I
don't know how far along they got.

Paul

**Comment by elsurudo** [comment]
Still no updates? Hard to believe there isn't a decent wizarding gem that works with Rails 3!

**Comment by paulschreiber** [comment]
On Feb 22, 2012, at 11:09 AM, Ernest Surudo wrote:

> Still no updates? Hard to believe there isn't a decent wizarding gem that works with Rails 3!

No updates. Our company is moving to Rails 3 next month, so were going to figure out/build something.

Paul

**Comment by elsurudo** [comment]
I've got immediate needs as well, so I've already started working on a solution. If it's any good, I'll be sure to pull it out into a gem and share when I have the time.

**Comment by paulschreiber** [comment]
On Feb 23, 2012, at 3:37 PM, Ernest Surudo wrote:

> I've got immediate needs as well, so I've already started working on a solution. If it's any good, I'll be sure to pull it out into a gem and share when I have the time. 

email me at paulschreiber@gmail.com — let’s talk about what you’re doing.

Paul

**Comment by manuelmeurer** [comment]
If you guys put a gem together, it would be great if you let us know in this thread! :) I built some wizards using the method Ryan Bates has shown in his Railscast about multi-step forms, but that is not the cleanest solution.

**Comment by elsurudo** [comment]
Yeah, I agree. I'm trying to keep my wizard logic in the controller, where it belongs. To me, it seems cleaner to have a separate action for each step.

I'm still really new to this, but will let you all know if something decent comes out of it!

**Comment by manuelmeurer** [comment]
Here's a new wizard gem: https://github.com/schneems/wicked
Blog post: http://schneems.com/post/18437886598/wizard-ify-your-rails-controllers-with-wicked

**Comment by elsurudo** [comment]
Thanks! Just found this one a couple of days ago myself. It's pretty well-written, but functionality is pretty basic. Overall, a good base to build something more complex on.
